custom-connectors/AzureADRiskyUsers/apiDefinition.swagger.json

```diff
@@ -0,0 +1,239 @@
+                  },
+                  "description": "userIds"
+                }
+              },
+              "default": {
+                "userIds": [
+                  "{{userID}}"
+                ]
+              }
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "204",
+            "schema": {}
+          }
+        }
+      }
+    },
+    "/v1.0/identityProtection/riskyUsers/{riskyUserId}": {
+      "get": {
+        "summary": "Get Risky user",
+        "description": "Get a single risky user",
+        "operationId": "GetRiskyUser",
+        "parameters": [
+          {
+            "name": "riskyUserId",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "description": "riskyUserId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "@odata.context": {
+                  "type": "string",
+                  "description": "@odata.context"
+                },
+                "id": {
+                  "type": "string",
+                  "description": "id"
+                },
+                "isDeleted": {
+                  "type": "boolean",
+                  "description": "isDeleted"
+                },
+                "isProcessing": {
+                  "type": "boolean",
+                  "description": "isProcessing"
+                },
+                "riskLevel": {
+                  "type": "string",
+                  "description": "riskLevel"
+                },
+                "riskState": {
+                  "type": "string",
+                  "description": "riskState"
+                },
+                "riskDetail": {
+                  "type": "string",
+                  "description": "riskDetail"
+                },
+                "riskLastUpdatedDateTime": {
+                  "type": "string",
+                  "description": "riskLastUpdatedDateTime"
+                },
+                "userDisplayName": {
+                  "type": "string",
+                  "description": "userDisplayName"
+                },
+                "userPrincipalName": {
+                  "type": "string",
+                  "description": "userPrincipalName"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "undefined": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://login.windows.net/common/oauth2/authorize",
+      "tokenUrl": "https://login.windows.net/common/oauth2/authorize",
+      "scopes": {}
+    }
+  },
+  "security": [
+    {
+      "undefined": []
+    }
+  ],
+  "tags": []
+}
+    "publisher": "Jan Bakker",
```

Comment:
The publisher and Stackowner properties are not required in the swagger file, please remove them